omod/src/main/webapp/fragments/field/datetimepicker.gsp

```diff
@@ -11,8 +11,14 @@
 
     def dateStringFormat
     def dateISOFormatted
+    def dateStringFormat2
```

Comment:
@mogoodrich should this be named something like `fallbackDateStringFormat` then?

Reply:
Yeah, @mks-d  @samuelmale I would definitely support a less generic variable name as well

Reply:
Sure, makes sense to me as well